VideoWeb/VideoWeb/Helpers/ParticipantTilePositionHelper.cs

```diff
@@ -18,9 +18,11 @@ public static void AssignTilePositions(IEnumerable<ParticipantResponse> particip
         public static string GetTiledDisplayName(ParticipantResponse participant)
         {
             var prefix = "";
+            var heartbeatMode = "NO_HEARTBEAT";
```

Comment:
declare the NO_HEARTBEAT and HEARTBEAT as consts either in this file, or in a separate file so you could use that to reference in your unit test